tests/bears/BearTest.py

```diff
@@ -129,6 +140,24 @@ def test_bad_bear(self):
         # debug message contains custom content, dont test this here
         self.queue.get()
 
+    def test_LocalBear(self):  # Fails because of no run() implementation
```

Comment:
I would place the comment before `self.check_message(LOG_LEVEL.WARNING, ...)`, it somehow confuses me when I read it each time... sounds like the test case will always fail (which shouldn't happen, right :P)